server/src/Database.hs

```diff
@@ -139,13 +149,13 @@ getCredentialEntryById (Transaction conn) (M.CredentialId credentialId) = do
     [entry] -> pure $ Just $ toCredentialEntry entry
     _ -> fail "Unreachable: attested_credential_data.id has a unique index."
 
-getCredentialEntriesByUserId :: Transaction -> M.UserHandle -> IO [CredentialEntry]
-getCredentialEntriesByUserId (Transaction conn) (M.UserHandle userId) = do
+getCredentialEntriesByUserName :: Transaction -> M.UserAccountName -> IO [CredentialEntry]
```

Comment:
This function should probably be called `getCredentialEntriesByUser` to align with `userExists` or `getCredentialEntriesByUserAccountName` to align with the type it receives.